src/workflows/utils/labels.config

```diff
@@ -21,20 +21,17 @@ process {
   withLabel: veryhighmem { memory = { get_memory( 75.GB * task.attempt ) } }
 
   // Disk space
-  // Nextflow apparently can't handle empty directives, i.e.
-  // withLabel: lowdisk {}
-  // so for that reason we have to add a dummy directive
   withLabel: lowdisk {
-    dummyDirective = "dummyValue"
+    disk = {process.disk ? process.disk : null}
```

Comment:
To check if this works, I used this snippet:

```
  process {
    disk = "100.GB"
    // Disk space
    withLabel: lowdisk {
      disk = {process.disk ? "anothervalidvalue" : null}
    }
  }
```
Which causes a workflow to fail with:
```
Caused by:
  Not a valid disk value: anothervalidvalue
```


However, just (no `process.disk`): 

```
  process {
    // Disk space
    withLabel: lowdisk {
      disk = {process.disk ? "anothervalidvalue" : null}
    }
  }
```
Does not cause a workflow to fail.

The location of `process.disk` also does not matter. This also fails: 

```
  process {
    // Disk space
    withLabel: lowdisk {
      disk = {process.disk ? "anothervalidvalue" : null}
    }
    disk = "100.GB"
  }
```